packages/clay-toolbar/stories/index.tsx

```diff
@@ -211,4 +212,97 @@ storiesOf('Components|ClayToolbar', module)
 				</ClayToolbar.Nav>
 			</ClayLayout.ContainerFluid>
 		</ClayToolbar>
+	))
+	.add('Upper Toolbar', () => (
+		<ClayToolbar displayType="light">
+			<ClayLayout.ContainerFluid>
+				<ClayToolbar.Nav>
+					<ClayToolbar.Item expand>
+						<span className="text-left">
+							{'Workshop_idea_generation.pdf'}
+						</span>
+					</ClayToolbar.Item>
+
+					<ClayToolbar.Item>
+						<ClayButton.Group>
+							<ClayButtonWithIcon
+								displayType="secondary"
+								small
+								spritemap={spritemap}
+								symbol="angle-left"
+							/>
+							<ClayButtonWithIcon
+								displayType="secondary"
+								small
+								spritemap={spritemap}
+								symbol="angle-right"
+							/>
+						</ClayButton.Group>
+					</ClayToolbar.Item>
+
+					<ClayToolbar.Item>
+						<ClayButton displayType="secondary" small>
+							{'Share'}
+						</ClayButton>
+					</ClayToolbar.Item>
+
+					<ClayToolbar.Item>
+						<ClayButton small>
+							<span className="inline-item inline-item-before">
+								<ClayIcon
+									spritemap={spritemap}
+									symbol="download"
+								/>
+							</span>
+
+							{'Download'}
+						</ClayButton>
+					</ClayToolbar.Item>
+
+					<ClayToolbar.Item>
+						<ClayButtonWithIcon
+							displayType="unstyled"
+							small
+							spritemap={spritemap}
+							symbol="ellipsis-v"
+						/>
+					</ClayToolbar.Item>
+				</ClayToolbar.Nav>
+			</ClayLayout.ContainerFluid>
+		</ClayToolbar>
+	))
+	.add('Upper Toolbar #2', () => (
```

Comment:
We don't need this example. This is covered by Navbar (Application Bar). I think having this example would mislead users to using the wrong component.

Reply:
Alright, great, that was my main point of confusion. Thanks for clarifying